mitodl/mitxpro
# #147 Product Page: Courses in this Program

#### Pre-Flight checklist


#### What are the relevant tickets?
Closes #147 

#### What's this PR do?
Adds a "Courses in this Program" subpage type.

#### How should this be manually tested?
Add a "Courses in this Program" type subpage to a Program Page and this carousel should appear on all the Course Pages of courses included in the program.

#### Screenshots (if appropriate)
(Optional)
![Screenshot from 2019-05-07 17-28-52](https://user-images.githubusercontent.com/45350418/57299455-09c5dc00-70ee-11e9-81ce-1787c1279b56.png)
![Screenshot from 2019-05-07 17-29-22](https://user-images.githubusercontent.com/45350418/57299458-0df1f980-70ee-11e9-82bd-4cdae443888e.png)
